docs/getting-started-guides/kubeadm.md

```diff
@@ -42,16 +42,6 @@ You will install the following packages on all the machines:
 
 For each host in turn:
 
-<!--
-    # curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-    # cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
-    deb http://packages.cloud.google.com/apt kubernetes-xenial main
-    EOF
-    # apt-get update
-    # apt-get install -y kubeadm docker.io§
--->
```

Comment:
This is an unrelated change, but I thought I'd just clean up a useless comment while I was here.